Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ccl/dccwalletinfo/update/DccWalletInfoUpdateTask.kt

```diff
@@ -36,20 +34,18 @@ class DccWalletInfoUpdateTask @Inject constructor(
         }
 
         dccWalletInfoCleaner.clean()
-
         return object : Task.Result {}
     }
 
     override suspend fun cancel() = Unit
 
     data class Arguments(
         val dccWalletInfoUpdateTriggerType: DccWalletInfoUpdateTriggerType,
-        val startDelay: Long = 1_000L
     ) : Task.Arguments
 
-    sealed class DccWalletInfoUpdateTriggerType {
-        object TriggeredAfterCertificateChange : DccWalletInfoUpdateTriggerType()
-        data class TriggeredAfterConfigUpdate(val configurationChanged: Boolean) : DccWalletInfoUpdateTriggerType()
+    sealed interface DccWalletInfoUpdateTriggerType {
```

Comment:
`TriggerType` is now also not needed anymore. We currently only use this Tasks for Config Updates. But we can do this clean up in a subsequent PR as well.

Reply:
Yes, but I want to keep the changes minimal, as I mentioned in the daily I would like to merge this merge and do the other changes in another PR ,since it is not really to fixing / optimising the performance

Reply:
fine for me!